libpkg/clean_cache.c

```diff
@@ -48,7 +48,7 @@ rm_rf(int basefd, const char *path)
 
 		dirfd = openat(basefd, path, O_DIRECTORY|O_CLOEXEC);
 		if (dirfd == -1) {
-			pkg_emit_errno("openat", path);
+			pkg_emit_errno("openat", __func__);
```

Comment:
Well, yes.  However as a user of pkg(8) trying to debug why my pkg install didn't work, I'd find it more relevant to see the filename here.   That's going to be a lot more useful in fixing whatever the problem  was than the internal function name.